src/data/partners.json

```diff
@@ -242,6 +242,12 @@
       "slug": "tech",
       "image": "joliberlin.jpg",
       "link": "https://www.joliberlin.com"
+    },
+       {
```

Comment:
watch out here. has to be aligned. will fix it

Reply:
forgot about it :( good call!